apps/lib/signer/signer.go

```diff
@@ -122,10 +137,10 @@ func (s *StarkSigner) SignPublisherPrice(publishTimestamp int64, asset string, q
 	assetHex := hex.EncodeToString([]byte(asset))
 	assetHexPadded := assetHex
 	if len(assetHex) < 34 {
-		assetHexPadded = "0x" + assetHex + strings.Repeat("0", 32-len(assetHex))
+		assetHexPadded = assetHex + strings.Repeat("0", 32-len(assetHex))
```

Comment:
I think this was a potential bug before? we were adding `0x` only if the input string was length <34. adding it outside of the if now.